Ashish013 patch 1

Ashish013 opened the pull request:
Here are few modifications in the script!!
Added the health monitoring module and tweaked the code for some bugs and all!!

prashantsengar commented:
Good work @Ashish013. Can you please explain in detail what exactly did you add or modify? Try including screenshots too.

Ashish013 commented:
Added the functionality using the module named pchealth.py to give stats on Pc health and storage info as show in the screenshot attached below

![Screenshot (41)](https://user-images.githubusercontent.com/56002521/70451696-d7d58780-1acb-11ea-87b4-59a080b83d47.png)

It can also detect CPU temperature and CPU Fan Speed( Not available on Windows) provided they are supported by the system and most importantly by the OS.

Removed a bug where the folder are created irrespective of whether the particular 
file type (Docs, Audio,etc) exists in the current directory or not resulting in empty folders 
many a times

![Screenshot (42)](https://user-images.githubusercontent.com/56002521/70451996-63e7af00-1acc-11ea-8520-7afc372b6418.png)

In the screenshot attached above the "Docs" folder is not created as no such file of Doc's
datatype (.xlsx, .doc ,etc) exists  in the current directory
